src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -659,7 +659,41 @@ paths:
         connectionType: VPC_LINK
         connectionId: '${vpc_connection_id}'
         httpMethod: POST
-
+    patch:
+      summary: Updates specific details within an Anonymous Elector Document (AED).
+      description: Updates specific details (e.g. the elector's phone number) within an Anonymous Elector Document (AED). Note that if more than one AED has been issued for an application, then all of them will be updated.
```

Comment:
Minor: Updates specific `contact` details of an AED...

Reply:
I don't want to be too specific in case other fields are added in future.